pkg/cmd/cli/cmd/newapp.go

```diff
@@ -44,6 +45,9 @@ Examples:
 	# Use a MySQL image in a private registry to create an app
 	$ %[1]s new-app myregistry.com/mycompany/mysql
 
+	# Create an application from the remote repository using the specified label
+	$ %[1]s new-app https://github.com/openshift/ruby-hello-world -l FOO=BAR
```

Comment:
Labels aren't env vars, use lowercase.  Don't use "foo" or "bar" in help for end users, use a context appropriate name.  Here, "name=hello-world" would be appropriate.